assets/manifest_template.toml

```diff
@@ -1,5 +1,5 @@
 #
```

Comment:
blocking: needs tests for
- flox activate && eval "$(flox activate)" - profile gets run a second time
  - probably the same for flox activate && flox activate -- cmd
- flox activate (sets VAR=1) && export VAR=2 && eval "$(flox activate)" - VAR is set back to 1
- flox activate (unsets some var) && export VAR && eval "$(flox activate)" - unsets var
- eval "$(flox activate)" && eval "$(flox activate)" - hook does not run a second time
   - probably the same for flox activate && flox activate -- cmd

Reply:
Fixed